client/index.html

```diff
@@ -1,27 +1,81 @@
 <!DOCTYPE html>
 <html lang="ko">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>배민 로그인</title>
-    <link rel="stylesheet" href="./assets/css/index.css">
-    <link rel="stylesheet" href="./assets/css/login.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>배민 로그인</title>
+  <link rel="stylesheet" href="./assets/css/index.css">
+  <link rel="stylesheet" href="./assets/css/main.css">
 </head>
 <body>
-    <header class="header-layout">
-        <button class="close-btn">
-            <img src="./assets/icons/x.svg" alt="close-btn"/>
-        </button>
-    </header>
-    <main class="login-page-layout">
-        <form class="login-form">
-            <input class="woowa-input login-form__id" placeholder="아이디를 입력해주세요"></input>
-            <input class="woowa-input login-form__pw" type="password" placeholder="비밀번호를 입력해주세요"></input>
-            <input class="woowa-btn login-form__submit-btn" type="submit" value="로그인"></input>
-        </form>
-        <button class="woowa-btn signup-btn" disabled>회원가입</button>
-    </main>
-    <script src="./assets/js/index.js"></script>
+  <header class="header-layout">
+      <div class="header-layout__title">My 배민</div>
+  </header>
+  <main class="main-page-layout">
+    <section class="main-section">
+      <div class="main-banner"></div>
```

Comment:
썸네일을 위한 공간으로 figure 태그도 나쁘지 않아보입니다

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure

```suggestion
      <figure class="main-banner"></figure>
```

Reply:
test reply